source/backend/routes/users.js

```diff
@@ -86,4 +86,20 @@ router.delete('/:savedRecipeId', async (req, res) => {
   }
 });
 
+/* POST /user/completedRecipes */
+router.post('/completedRecipes/:recipeId', async (req, res) => {
+  try {
+    const { userId } = req.userInfo;
+    const { recipeId } = req.params;
```

Comment:
Now I am thinking about this, we need to do a check first to see if this user already completed the recipe, if so, return a 401 forbidden telling user that this recipe is already completed. 

Same idea as the signup, do a get with the userId and recipeId in the completeRecipes table, if length > 0 return 401

Reply:
Fixed! Double check to make sure it's good